<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gitautodeploy_config_MainWindow</class>
 <widget class="QMainWindow" name="gitautodeploy_config_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Git-auto-deploy preference</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Myriad Pro Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Git-auto-deploy setting</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="main_tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="common_tab">
       <attribute name="title">
        <string>Common</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_service_status">
          <property name="title">
           <string>services</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,0">
           <item row="0" column="1">
            <widget class="QLabel" name="label_service_name">
             <property name="text">
              <string>Service name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_service_status">
             <property name="text">
              <string>Status</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="service_gitautodeploy_start_pushButton">
             <property name="text">
              <string>Start / Stop</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="service_gitautodeploy_pid">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="service_gitautodeploy_log_pushButton">
             <property name="text">
              <string>Logs</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="service_label_gitautodeploy">
             <property name="text">
              <string>gitautodeploy</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_service_pid">
             <property name="text">
              <string>PID(s)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="service_gitautodeploy_status">
             <property name="text">
              <string>stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_service_port">
             <property name="text">
              <string>Port(s)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="service_gitautodeploy_port">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="service_ngrok_status">
             <property name="text">
              <string>stop</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="service_label_ngrok">
             <property name="text">
              <string>ngrok</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="service_ngrok_pid">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="service_ngrok_port">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QPushButton" name="service_ngrok_start_pushButton">
             <property name="text">
              <string>Start / Stop</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QPushButton" name="service_ngrok_log_pushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Logs</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4" colspan="2">
            <widget class="QLabel" name="label_service_action">
             <property name="text">
              <string>Action</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Config</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_pid_filepath"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_browse_pid_filepath">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Daemon mode</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="checkBox_daemon_mode_active">
             <property name="text">
              <string>Active</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Logs file path</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="lineEdit_log_filepath"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_browse_log_filepath">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>log-level</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QComboBox" name="comboBox_log_level">
               <item>
                <property name="text">
                 <string>INFO</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>WARNING</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ERROR</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>CRITICAL</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>DEBUG</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>NOTSET</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>PID file path</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_pre_post_deploy_script">
             <property name="title">
              <string>Global deploy script</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_Pre_deploy_script">
                <property name="text">
                 <string>Pre-deploy script</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_Post_deploy_script">
                <property name="text">
                 <string>Post-deploy script</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_predeploy_script"/>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_postdeploy_script"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_httpHost">
             <property name="text">
              <string>HTTP Host</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0,1">
             <item>
              <widget class="QLineEdit" name="lineEdit_http_host"/>
             </item>
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Port</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_http_port"/>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_http_enable">
               <property name="text">
                <string>Enable</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_httpsHost">
             <property name="text">
              <string>HTTPS Host</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,0,0,1">
             <item>
              <widget class="QLineEdit" name="lineEdit_https_host"/>
             </item>
             <item>
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Port</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_https_port"/>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_https_enable">
               <property name="text">
                <string>Enable</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="webui_tab">
       <attribute name="title">
        <string>Web UI</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Setting</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBox_enable_webui">
             <property name="text">
              <string>Enable Web UI</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_enable_webui_auth">
             <property name="text">
              <string>Enable auth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_enable_webui_https">
             <property name="text">
              <string>Require https (secure connect)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Web UI whilte list</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QListWidget" name="listWidget_webui_whitelist"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Login</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_webui_auth_username"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_webui_auth_password">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>SSL Credential</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Key path</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Cert path</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEdit_sll_key_path"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_browse_key">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="lineEdit_sll_cert_path"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_browse_cert">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gitrepo_tab">
       <attribute name="title">
        <string>Repositories</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget_repositories">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Repository name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Current branch</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last commit date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Edit filter</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_repository_add">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;add repository&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_repository_delete">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;remove repository&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_repository_pull">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;pull repository&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>↓</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="submit_buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Abort|QDialogButtonBox::Apply|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
